third_party/move/move-vm/integration-tests/src/tests/bad_storage_tests.rs

```diff
@@ -567,7 +565,7 @@ fn test_storage_returns_bogus_error_when_loading_module() {
             )
             .unwrap_err();
 
-        assert_eq!(err.status_type(), StatusType::InvariantViolation);
+        assert_eq!(err.major_status(), *error_code);
```

Comment:
@vgao1996 @runtian-zhou @wrwg do you know what this test is supposed to test? I would assume that error types are preserved? Or that we map everything to invariant violation?